pubsublite/manager.go

```diff
@@ -240,3 +252,93 @@ func (m *Manager) DeleteSubscription(ctx context.Context, subscription string) e
 	logger.Info("deleted pubsublite subscription")
 	return nil
 }
+
+// MonitorConsumerLag registers a callback with OpenTelemetry
+// to measure consumer group lag for the given topics.
+func (m *Manager) MonitorConsumerLag(topics ...string) (metric.Registration, error) {
```

Comment:
It takes `string` for topic for consistency as other public methods under pubsublite manager also take a string topic, as opposed to apmqueue.Topic in Kafka manager.

Reply:
Actually we should probably accept "topics-consumers", i.e. a list of (topic, consumer) pairs. Topics may be prefixed for namespacing/multi-tenancy, whereas consumers will probably not differ across tenants.